File: C/3141.c
#include <stdio.h>

int main(){

    char nome[51];
    int dia, mes, ano, diaNasc, mesNasc, anoNasc;

    scanf("%[^\n]s", nome);
    scanf("%d/%d/%d", &dia, &mes, &ano);
    scanf("%d/%d/%d", &diaNasc, &mesNasc, &anoNasc);

    if(dia == diaNasc && mes == mesNasc){
        printf("Feliz aniversario!\n");
    }

    int d = (mes-mesNasc) * 31 + dia - diaNasc;

    if (d >= 0){
        printf("Voce tem %d anos %s.\n", (ano - anoNasc), nome);
    }else{
        printf("Voce tem %d anos %s.\n", (ano - anoNasc - 1), nome);
    }

    return 0;
}

File: Python/3141.py
nome = input()

dia, mes, ano = map(int, input().split("/"))
diaNasc, mesNasc, anoNasc = map(int, input().split("/"))

if dia == diaNasc and mes == mesNasc:
    print("Feliz aniversario!")

d = (mes - mesNasc) * 31 + dia - diaNasc

if d >= 0:
    print(f"Voce tem {ano - anoNasc} anos {nome}.")
else:
    print(f"Voce tem {ano - anoNasc - 1} anos {nome}.")
